numpy/lib/twodim_base.py

```diff
@@ -172,21 +245,33 @@ def rot90(m, k=1):
     >>> np.rot90(m, 2)
     array([[4, 3],
            [2, 1]])
+    >>> m = np.arange(8).reshape((2,2,2))
+    >>> np.rot90(m, 1, (1,2))
+    array([[[1, 3],
+            [0, 2]],
+
+          [[5, 7],
+           [4, 6]]])
 
     """
     m = asanyarray(m)
+
     if m.ndim < 2:
         raise ValueError("Input must >= 2-d.")
```

Comment:
added A) check for len(axes) in [1a52123](https://github.com/erensezener/numpy/commit/1a521230bc2ccce71e3bbee722364206eb090aa9)

added B) removed check for ```ndim >= 2``` in [e5b5b96](https://github.com/erensezener/numpy/commit/e5b5b96922d74fd74c8ee6ebe25e0a052808955e) and added checking for ndim >=2  and a test for it in [aa0d4bf](https://github.com/erensezener/numpy/commit/aa0d4bf2ad3d3a0c29b50c817fda2c6a04348db6)

added raise for same axes and a test for it in [83147ee](https://github.com/erensezener/numpy/commit/83147eee7140e211d79b9166bf730878186ce6bf)